cli/tools/repl/session.rs

```diff
@@ -82,67 +84,59 @@ fn comment_source_to_position_range(
   }
 }
 
-/// We store functions used in the repl on this object because
-/// the user might modify the `Deno` global or delete it outright.
-pub static REPL_INTERNALS_NAME: Lazy<String> = Lazy::new(|| {
-  let now = std::time::SystemTime::now();
-  let seconds = now
-    .duration_since(std::time::SystemTime::UNIX_EPOCH)
-    .unwrap()
-    .as_secs();
-  // use a changing variable name to make it hard to depend on this
-  format!("__DENO_REPL_INTERNALS_{seconds}__")
-});
-
+thread_local! {
```

Comment:
I'm not a fan of using a `thread_local` here - consider storing this object id on `ReplSession` object